### ISSUE TITLE
`nix-shell -A shells.ghc --arg withHoogle true` does not work

### ISSUE THREAD

**Issue by Aearnus** [obsidiansystems/obelisk#624]
Hello!

The README specifies `nix-shell -A shells.ghc --arg withHoogle true` as the command to run in order to host a local Hoogle server for your Obelisk project. However, that produces the following error: 

```
error: attribute 'shells' in selection path 'shells.ghc' not found
```

Additionally, passing `withHoogle` to the default `nix-shell` as `nix-shell --arg withHoogle true` starts a shell but does not make Hoogle available.

Reproduction environment
===
Current head, project generated by `ob init`

Workaround
===
Change `default.nix` to contain `withHoogle`:

```
{ obelisk ? import ./.obelisk/impl {
    system = builtins.currentSystem;
    iosSdkVersion = "10.2";
    # You must accept the Android Software Development Kit License Agreement at
    # https://developer.android.com/studio/terms in order to build Android apps.
    # Uncomment and set this to `true` to indicate your acceptance:
    # config.android_sdk.accept_license = false;
  }
}:
with obelisk;
project ./. ({ ... }: {
  android.applicationId = "systems.obsidian.obelisk.examples.minimal";
  android.displayName = "Obelisk Minimal Example";
  ios.bundleIdentifier = "systems.obsidian.obelisk.examples.minimal";
  ios.bundleName = "Obelisk Minimal Example";
  withHoogle = true;
})
```

**Comment by endgame** [comment]
See also #68.

**Comment by 3noch** [comment]
Thanks for reporting! We plan to fix this "properly" soon!

**Comment by 3noch** [comment]
@endgame @Aearnus The "proper" fix is on its way :smile:  https://github.com/obsidiansystems/obelisk/pull/628

**Comment by Aearnus** [comment]
> @endgame @Aearnus The "proper" fix is on its way  #628

Looks awesome! Thank you @3noch.
